src/main/java/com/study/stockmanagementstudycase/controller/StockController.java

```diff
@@ -109,4 +103,19 @@ public ResponseEntity<List<WareHouseStockResponse>> getWareHouseStocks(
 
         return ResponseEntity.ok(wareHouseStockResponses);
     }
+
+    @PostMapping("/{stockId}/stockUpdate")
```

Comment:
@PutMapping kullanabiliriz. Dolayısıyla endpoint'in yolu için de "/{stockId}" yeterli olacaktır.